parquet_file/src/lib.rs

```diff
@@ -12,3 +12,4 @@ pub mod chunk;
 pub mod metadata;
 pub mod storage;
 pub mod table;
+pub mod utils;
```

Comment:
If these helpers should only be used for tests, I usually mark them as test only to keep them out of production code:

```suggestion
#[cfg(test)]
pub mod utils;
```

This means they can only be imported when running tests - it depends how you want them to be used though!

Reply:
Yes, keep it test only for now